modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -14,6 +14,470 @@
+=== Web Embed Mode
+
+The Web UI now provides an Embed Mode for easier integration into other applications. The Embed Mode allows external applications to integrate the Web UI directly. This means that users can now access and interact with Infinite Scale within the context of other applications.
+
+Example: Imagine you're using a chat application and want to send a sharing link. With the Embed Mode, you can open the Web UI in a file picker-like interface, select files from Infinite Scale, and share them without ever having to leave the chat tab.
+
+The Embed Mode streamlines workflows and eliminates the need to switch between different applications to manage files.
+
+[discrete]
+=== Focused "New" Menu
+
+The `New` menu items have been reorganized with the most frequently used items placed at the top. This rearrangement is based on user feedback, ensuring that the most important filetypes are readily accessible. In our commitment to Open Source and universal accessibility, we've revised the wording for Document labels to be more vendor-neutral. This change reflects our dedication to providing a user-friendly interface that caters to a universal user base.
+
+[discrete]
+=== Open the Sidebar From Everywhere
+We have relocated the button to open the sidebar, positioning it now in the global top bar. This move allows users to access the right sidebar not just in the Files app, but also in a variety of other applications. This change provides a way for integrating features like file details and sharing options into other applications, beyond the Files app.
+
+[discrete]
+=== 400% Faster Upload Preparation Time
+
+In the latest update, we've implemented significant improvements in the efficiency of folder tree creation during file uploads. By optimizing the process to run asynchronously and reducing the number of PROPFIND requests on nested folders, we've managed to substantially speed up this operation. In a test scenario with a folder containing 155 subfolders, the time to create these folders has been reduced from 20-30 seconds to just 5-7 seconds. While this duration may still be noticeable, it represents a considerable improvement, especially considering the limitations of client-side operations.
+
+Additionally, we've massively enhanced the upload preparation time. Rather than setting file data individually for each file, we now collect all necessary data first and then apply it in a single batch using Uppy's setState method. This approach streamlines the upload process, making it much quicker and more efficient for users.
+
+[discrete]
+=== Thumbnail Generation Using Image Processors
+
+When thumbnail creation is requested by the WebUI, the format can now be changed as part of the creation process. Previously images were always scaled to fit the given frame where it could happen, that images were cropped to fit making them often hard to identify. By defining a processor via the API, images can now be scaled to best fit a given frame.
```

Comment:
```suggestion
When thumbnail creation is requested by the WebUI, the format can now be changed as part of the creation process. Previously images were always scaled to fit the given frame. In the process it could happen that images were cropped to fit, making them often hard to identify. By defining a processor via the API, images can now be scaled to best fit a given frame.
```